conda/recipes/ucx-py/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = 'ucx-py' %}
+{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') + environ.get('VERSION_SUFFIX', '') %}
+{% set py_version = environ.get('CONDA_PY', '38') %}
+{% set cuda_version = '.'.join(environ.get('CUDA', '11.2').split('.')[:2]) %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  path: ../../..
+
+build:
+  number: {{ GIT_DESCRIBE_NUMBER }}
+  string: py{{ py_version }}_g{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
+  script_env:
+    - VERSION_SUFFIX
+    - CC
+    - CXX
+    - CUDAHOSTCXX
+  script:
+    - {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python
+    - pip
+    - cython >=0.29.14,<3.0.0a0
+    - libhwloc >=2.3.0,<2.4.0a0
+    - ucx
+  run:
+    - python
+    - numpy
+    - psutil
+    - ucx
+  run_constrained:
+    - pynvml >=11.4.1
```

Comment:
Based on offline discussion with Peter, we think 11 is a minimum requirement, but there were some fixes in the later versions that we may also need. So have added 11.4.1 as a minimum here